### PR TITLE
Adapts import paths to be explicit

### LINKED ISSUE #1180: imports from `../index.js`
We're currently using import syntax like this, when importing react-geo components 
 (ZoomButton):

```js
import  {
  SimpleButton
} from '../../index';
```

If I'm not wrong this means that, even if i import a specific component (from outside of react-geo) I'll import the whole library. e.g.:

```js
import ZoomButton from '@terrestris/react-geo/dist/Button/ZoomButton/ZoomButton';
```

imports everything from index.js.

### PR DESCRIPTION
**Pull request by KaiVolland**
### Description:
This adapts the import paths inside the react-geo components to be more precise.

closes #1180 

Thx @mholthausen for the help

### CODE REVIEW

File: src/Button/MeasureButton/MeasureButton.spec.jsx [hunk 1]
```diff
@@ -210,7 +210,7 @@ describe('<MeasureButton />', () => {
         expect(createHelpTooltipSpy).toHaveBeenCalledTimes(1);
         expect(createMeasureTooltipSpy).toHaveBeenCalledTimes(1);
 
-        jest.resetAllMocks();
+        // jest.resetAllMocks();
```
Comment (dnlkoch) [inline src/Button/MeasureButton/MeasureButton.spec.jsx]: Covered by #1182.

File: src/Button/MeasureButton/MeasureButton.spec.jsx [hunk 2]
```diff
@@ -259,7 +259,7 @@ describe('<MeasureButton />', () => {
         expect(createHelpTooltipSpy).toHaveBeenCalledTimes(1);
         expect(clearSpy).toHaveBeenCalledTimes(1);
 
-        jest.resetAllMocks();
+        // jest.resetAllMocks();
```
Comment (dnlkoch) [inline src/Button/MeasureButton/MeasureButton.spec.jsx]: Covered by #1182.

File: src/Button/MeasureButton/MeasureButton.spec.jsx [hunk 3]
```diff
@@ -305,7 +305,7 @@ describe('<MeasureButton />', () => {
         const removeMeasureTooltipSpy = jest.spyOn(instance, 'removeMeasureTooltip');
         instance.onDrawEnd(mockEvt);
         expect(removeMeasureTooltipSpy).toHaveBeenCalledTimes(1);
-        jest.resetAllMocks();
+        // jest.resetAllMocks();
```
Comment (dnlkoch) [inline src/Button/MeasureButton/MeasureButton.spec.jsx]: Covered by #1182.

File: src/Button/MeasureButton/MeasureButton.spec.jsx [hunk 4]
```diff
@@ -335,7 +335,7 @@ describe('<MeasureButton />', () => {
         const createMeasureTooltipSpy = jest.spyOn(instance, 'createMeasureTooltip');
         instance.onDrawEnd(mockEvt);
         expect(createMeasureTooltipSpy).toHaveBeenCalledTimes(1);
-        jest.resetAllMocks();
+        // jest.resetAllMocks();
```
Comment (dnlkoch) [inline src/Button/MeasureButton/MeasureButton.spec.jsx]: Covered by #1182.

File: src/HigherOrderComponent/DropTargetMap/DropTargetMap.spec.jsx [hunk 5]
```diff
@@ -3,9 +3,9 @@ import React from 'react';
 
 import TestUtil from '../../Util/TestUtil';
 
-jest.unmock('../../index');
+// jest.unmock('../../index');
```
Comment (dnlkoch) [inline src/HigherOrderComponent/DropTargetMap/DropTargetMap.spec.jsx]: Is this intended?

File: src/Panel/Panel/Panel.spec.jsx [hunk 6]
```diff
@@ -61,7 +61,7 @@ describe('<Panel />', () => {
       expect(onEscSpy).toHaveBeenCalledTimes(1);
       expect(focusSpy).toHaveBeenCalledTimes(1);
       expect(element.className).toContain(document.activeElement.className);
-      jest.resetAllMocks();
+      // jest.resetAllMocks();
```
Comment (dnlkoch) [inline src/Panel/Panel/Panel.spec.jsx]: Covered by #1182.

File: src/HigherOrderComponent/DropTargetMap/DropTargetMap.spec.jsx (hunk 5 above)
Comment (KaiVolland) [reply src/HigherOrderComponent/DropTargetMap/DropTargetMap.spec.jsx]: Yes.

**Pull request merged**